### ISSUE TITLE
Spike: Determine work necessary to configure Google Analytics for Blacklight Catalog

### ISSUE THREAD

**Issue by lovinscari** [emory-libraries/blacklight-catalog#594]
Google Analytics should be configured for Blacklight Catalog in an effort to gather metrics and assess how the Blacklight Discovery Layer / Catalog is being used. Google Analytics were configured for DLP but some customizations may not be fully documented.

Acceptance Criteria:

- [x] Review documentation referenced in DLP Tickets [#189](https://app.zenhub.com/workspaces/digital-library-project-5bf484ae4b5806bc2bf6875b/issues/emory-libraries/dlp-lux/189) and [190](https://app.zenhub.com/workspaces/digital-library-project-5bf484ae4b5806bc2bf6875b/issues/emory-libraries/dlp-lux/190)

- [ ] If above documentation is found, work with product owner determine any changes needed for Google Analytics associated with Blacklight Catalog

- [ ] If work needed is less than 13 points, assignee should create new ticket(s) for work to be completed and enter into current sprint. 

- [ ] If work needed exceeds 13 points then create new tickets but consult with product owner to determine timing to implement the work.

**Comment by maxkadel** [comment]
Link to DCE documentation on google analytics - https://curationexperts.github.io/recipes/patterns/google_analytics.html . 

You can ignore the instructions about putting the `GA_TRACKING_CODE` in the `-cm` repository - basically add it to the environment the same way you'd add any other environment variable for this application.

**Comment by eporter23** [comment]
For Lux, we enabled a couple of custom events for aggregating views back to their parent collection and holding library. This may not be relevant to the catalog, but mentioning it in case we want to do something similar.

We also configured search query terms tracking (see[ Lux ticket 190](https://app.zenhub.com/workspaces/digital-library-project-5bf484ae4b5806bc2bf6875b/issues/emory-libraries/dlp-lux/190)).
